load_data.py

```diff
@@ -45,22 +46,105 @@ def load_config(contents, h5_filename, data):
 
     return data
 
+
+def load_pythTB_json(contents):
+    '''
+    read pythTB json file
+
+    Parameters
+    ----------
+    contents: json string from memory
+
+    Returns
+    -------
+    norb: int
+        number of orbitals
+    units: list of tuples
+        vectors spanning unit cell
+    hopping_dict: dict
+        dict of hopping tuples
+    '''
+    content_type, content_string = contents.split(',')
+    data_stream = base64.b64decode(content_string)
+    data = json.loads(data_stream)
+
+    lat = data['_lat']
+    hoppings = data['_hoppings']
+    site_energies = data['_site_energies']
+    norb = data['_norb']
+    units = []
+
+    # apparently if _dim_r <= 2 then an array doesnt need to be passed in the hoppings
+    if data['_dim_r'] <= 2:
+        raise Exception("The pyTB lattice must be greater than 2x2. ie _dim_r > 2")
+
+    # extract the lattice dimensions
+    for i in lat:
+        units.append(tuple(i))
+
+    # extract the hoppings
+    vectors = []  # list of np arrays
+    temp_dict = {}  # this dict is to keep track of instances of conjugate pair vectors
+    hopping_dict = {}  # this one that will be sent to make a triqs lattice object
+
+    for i in hoppings:
+        # not correct atm
+        t = i[0]
```

Comment:
This has to be fixed first. There can be off diagonal terms in the hoppings. Meaning that we have to store for each hopping a matrix norb x norb.